pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -250,7 +255,7 @@ func (pc *PodCache) update(ip string, key types.NamespacedName) {
 	}
 	pc.Unlock()
 
-	pc.proxyUpdates(ip)
+	pc.proxyUpdates(pod, false)
```

Comment:
if it comes here it is a new pod right? I think it may be easier to read if we change `proxyUpdate` to `proxyUpdates(pod *v1.Pod, **newpod** bool)` to flip the logic there.

Reply:
tried to clean it up, LMK if it helps

Reply:
Thank you. LGTM